src/components/CubeDraft.js

```diff
@@ -98,26 +98,31 @@ const CubeDraft = ({ draft, socket }) => {
     let status = 'inProgress';
     while (status === 'inProgress') {
       // we want to do this first as we don't want to spam the server
-      await new Promise((resolve) => setTimeout(resolve, 2000));
+      await new Promise((resolve) => {
+        setTimeout(resolve, 2000);
+      });
 
       try {
         const res = await callApi('/multiplayer/trybotpicks', {
           draft: draft.id,
         });
         if (res) {
-          let json = await res.json();
+          const json = await res.json();
           status = json.result;
 
           console.log(json);
 
-          if (json.picks == 0) {
-            await new Promise((resolve) => setTimeout(resolve, 5000));
+          if (json.picks === 0) {
+            await new Promise((resolve) => {
+              setTimeout(resolve, 5000);
+            });
+            // eslint-disable-next-line no-continue
             continue;
```

Comment:
But also, why is this `continue` even here? It's literally the last statement in the loop body (other than a `catch` block), so it does nothing?

Reply:
You're right, this `continue` statement can be removed.